src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { getUserById } from './utils/getUserById';
```

Comment:
[CODE STYLE] - According to the checklist, you should prepare data in one place (App component) and pass it to child components. Avoid using `import` keyword in non-root components for data that can be passed as props. Consider moving the `getUserById` logic into the App component and passing the necessary user data to the child components as props.